marker.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I see you created a custom svg for the marker. Nice. At some point, we may want to also change the marker to more resemble the drifter. @SLYpma could help with that? Let's discuss on Monday!